### PR TITLE
Support running on mac

### PR DESCRIPTION
**Pull request by rainbend**
image

image

### CODE REVIEW

File: pkg/components/common/common.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func (o *Operator) DownloadDaprClient(ctx context.Context, daprVersion string) e
 }
 
 func (o *Operator) InitDapr(ctx context.Context, daprVersion string) error {
-	cmd := fmt.Sprintf("dapr init -k --runtime-version %s", daprVersion)
+	cmd := fmt.Sprintf("dapr init -k --log-as-json --runtime-version %s", daprVersion)
```
Comment (rainbend) [inline pkg/components/common/common.go:106]: I just add `--log-as-json`, please help test @tpiperatgod :)

**Comment by rainbend** [comment]
Run the function:
image

**Comment by tpiperatgod** [comment]
LGTM

We need a Mac (with the Intel chip) volunteer to verify this change, thanks!

**Review by benjaminhuo — APPROVED** [review]

**Comment by benjaminhuo** [comment]
Merge it for now , I'll test later

Thanks @arugal !

**Pull request merged**